travis_ci.sh

```diff
@@ -38,7 +38,9 @@ if [ "$RUN_BUILD" == "true" ] || \
     travis_run ./environment_setup/setup_software.sh
 
     # Build the codebase
-    travis_run catkin_make
+    pushd src/thunderbots/
+    travis_run bazel build //... --compilation_mode=fastbuild
+    popd
```

Comment:
this is cool! never seen `pushd` and `popd` before, but this is super useful

Reply:
yeah same reaction when I first learned about it :)